.chloggen/fix-signalfxexporter-honorpassthrough.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: 'signalfxexporter'
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: 'eventclient should honor access_token_passthrough'
```

Comment:
eventclient is not very easy for users to read. Maybe: "Honor access_token_passthrough config option for sending events"?